dlrs/main/classes/RollupSummariesSelector.cls

```diff
@@ -90,41 +113,70 @@ public class RollupSummariesSelector {
     List<RollupSummaries.CalculationMode> calculationModes,
     Set<String> childObjectNames
   ) {
+    List<String> rollupSummaryNames = new List<String>();
+    for (RollupSummaries.CalculationMode cm : calculationModes) {
+      rollupSummaryNames.add(cm.name());
+    }
     List<RollupSummary> records = new List<RollupSummary>();
-    records = m_customObjectSelector.selectActiveByChildObject(
-      calculationModes,
-      childObjectNames
-    );
-    records.addAll(
-      m_customMetadataSelector.selectActiveByChildObject(
-        calculationModes,
-        childObjectNames
-      )
-    );
-    return records;
+    for (RollupSummary rs : this.allSummaries) {
+      if (
+        rs.Active &&
+        containsIgnoreCase(childObjectNames, rs.ChildObject) &&
+        containsIgnoreCase(rollupSummaryNames, rs.CalculationMode)
+      ) {
+        records.add(rs);
+      }
+    }
+    return sortSummaries(records, 'ParentObject__c', 'RelationshipField__c');
   }
 
   /**
    * Returns active lookup rollup summary definitions for the given rollup unique names
    **/
   public List<RollupSummary> selectActiveByUniqueName(Set<String> uniqueNames) {
     List<RollupSummary> records = new List<RollupSummary>();
-    records = m_customObjectSelector.selectActiveByUniqueName(uniqueNames);
-    records.addAll(
-      m_customMetadataSelector.selectActiveByUniqueName(uniqueNames)
-    );
+    for (RollupSummary rs : selectByUniqueName(uniqueNames)) {
+      if (rs.Active) {
+        records.add(rs);
+      }
+    }
     return records;
   }
 
+  /**
+   * Returns active lookup rollup summary definitions for the given rollup unique names
+   **/
+  public List<RollupSummary> selectByUniqueName(Set<String> uniqueNames) {
```

Comment:
Added to enable mapping the stub mdt records for scheduled items in RollupService.